dcicutils/es_utils.py

```diff
@@ -8,6 +10,64 @@
 logger.setLevel(logging.INFO)
 
 
+class ElasticSearchServiceClient:
+    """ Implements utilities for interacting with the Amazon ElasticSearch Service.
+        The idea is, for the production setup, we implement a hot/cold cluster configuration where
+        during the day (say 6 am to 8 pm EST) we run a larger cluster than at night/on
+        weekends. Foursight will implement this mechanism.
+     """
+    DEFAULT_HOT_MASTER_NODE_TYPE = 'c5.large.elasticsearch'
+    DEFAULT_HOT_MASTER_NODE_COUNT = 3
+    DEFAULT_HOT_DATA_NODE_TYPE = 'c5.2xlarge.elasticsearch'
+    DEFAULT_HOT_DATA_NODE_COUNT = 2
+
+    DEFAULT_COLD_MASTER_NODE_TYPE = 't2.small.elasticsearch'  # does not matter
+    DEFAULT_COLD_MASTER_NODE_COUNT = 0
+    DEFAULT_COLD_DATA_NODE_TYPE = 'c5.large.elasticsearch'
+    DEFAULT_COLD_DATA_NODE_COUNT = 2
```

Comment:
I don't see these values being used. Should they be the default for something?

Reply:
The idea is these could be used in foursight.